card/gem_mods/edit/set/all/editor.rb

```diff
@@ -46,6 +44,7 @@ def default_input_type
 
   # overridden by mods that provide rich text editors
   def rich_text_input
+    #FIXME: should select default differently, don't depend on an editor
     send "#{Cardio.config.rich_text_editor || :tinymce}_editor_input"
```

Comment:
This should eventually probably say `Cardio.config.rich_text_editor || :text_area`, because `text_areat_input` is the simple input defined in this mod. But for that to work we'd have to update existing decks and add `config.rich_text_editor = :ace` to config/application.rb.  We should probably be gathering those cases so that we can do them all at once.

Reply:
That should make it just a plain browser editor, no wysiwyg, right? (Done)